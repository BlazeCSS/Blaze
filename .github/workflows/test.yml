name: CI

on:
  push:
    branches:
      - master

jobs:
  build:
    name: Install, test and build
    
    runs-on: ubuntu-latest
    
    steps:
      - name: Checkout repo
        uses: actions/checkout@master

      - name: Install dependencies
        run: npm ci

      - name: Test
        run: npm run test

      - name: Build site
        run: npm run build
      
      - name: Switch to site dir
        run: cd ./packages/site

      - name: Deploy site
        uses: w9jds/firebase-action@master
        with:
          args: deploy --only hosting
        env:
          FIREBASE_TOKEN: ${{ secrets.FIREBASE_TOKEN }}
          PROJECT_ID: "default"
