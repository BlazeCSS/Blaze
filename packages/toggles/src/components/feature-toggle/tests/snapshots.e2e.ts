import { snapIt } from '../../../test';

const component = 'feature-toggle';

describe(component, () => {
  const snap = snapIt(component);

  describe('renders', () => {
    snap('nothing', '<feature-toggle>Hello, world</feature-toggle>');
  });
});
