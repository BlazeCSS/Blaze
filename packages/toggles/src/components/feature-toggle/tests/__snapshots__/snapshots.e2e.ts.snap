// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`feature-toggle renders nothing 1`] = `"<feature-toggle class=\\"hydrated\\"></feature-toggle>"`;
